plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py

```diff
@@ -25,26 +25,24 @@
 
 from pxr import Sdf, Usd
 
+import fixturesUtils
+
+
 class TestLayerManagerSerialisation(unittest.TestCase):
     """Test cases for layer manager serialisation and deserialisation"""
 
     @classmethod
     def setUpClass(cls):
-        # Reset the output directory
-        cls._outputPath = os.path.join(os.path.abspath('.'), os.path.splitext(__file__)[0] + "Output")
-        if os.path.exists(cls._outputPath):
-            shutil.rmtree(cls._outputPath)
-        os.mkdir(cls._outputPath)
-        cls._mayaFilePath = os.path.join(cls._outputPath, 'LayerManagerSerialisation.ma')
-        cls._usdFilePath = os.path.join(cls._outputPath, 'LayerManagerSerialisation.usda')
-
-        # Load the plugin
-        cmds.loadPlugin("AL_USDMayaPlugin", quiet=True)
+        # Load the plugin and setup test output
+        _ = fixturesUtils.setUpClass(__file__, pluginName="AL_USDMayaPlugin")
```

Comment:
Why assign the return value to `_` if you don't use it? I would prefer either a descriptive name of the variable or simply do not store return value.

Reply:
Yeah sorry about that. Bad habit I've imported from Ruby or Lua probably. I'll just remove it.